src/Files.Uwp/Strings/en-US/Resources.resw

```diff
@@ -2786,6 +2783,15 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="ReviewFilesContent" xml:space="preserve">
     <value>Would you like to review Files?</value>
   </data>
+  <data name="SetAsBackgroundFlyout" xml:space="preserve">
+    <value>Set as background</value>
+  </data>
+  <data name="SetAsLockscreen" xml:space="preserve">
+    <value>Set as lockscreen</value>
+  </data>
+  <data name="SetAsBackground" xml:space="preserve">
+    <value>Set as desktop background</value>
```

Comment:
Are these still being used?

Reply:
Ah, are these for the right click menu?

Reply:
I believe so

Reply:
We should bring the right click menu in line with the toolbar but this is fine for now.

Reply:
It's next on the list.